src/ddsrt/src/sockets/posix/socket.c

```diff
@@ -32,9 +32,9 @@
 #include <fcntl.h>
 #endif
 
-#ifdef __APPLE__
+#ifdef __APPLE__ || __FreeBSD__
```

Comment:
This should be changed to: `#if defined(__APPLE__) || defined(__FreeBSD__)`

Reply:
Incorporated.  Thanks!